DigitalLearningSolutions.Web/ViewModels/Home/LearningContentSummary.cs

```diff
@@ -0,0 +1,20 @@
+﻿namespace DigitalLearningSolutions.Web.ViewModels.Home
+{
+    using DigitalLearningSolutions.Data.Models.Common;
+
+    public class LearningContentSummary
+    {
+        public readonly int ID;
+        public readonly string Brand;
+        public readonly string? Description;
+        public readonly byte[]? Image;
+
+        public LearningContentSummary(BrandDetail brand)
+        {
+            this.ID = brand.BrandID;

```

Comment:
It's a little confusing, but in the codebase we have a custom of writing Id with the "d" lowercase. Example: "Id" and "BrandId". In views we display the word "ID" with the "D" uppercase, but in code we use camel case.